backend/db/manage_db.py

```diff
@@ -1 +1,83 @@
-# TODO create the database for the app
+from flask import Flask
+from sqlalchemy import create_engine  
+from sqlalchemy import Column, String, Float, Boolean
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.ext.declarative import declarative_base  
+from sqlalchemy.orm import sessionmaker
+
+from flask_script import Manager
+from flask_migrate import Migrate, MigrateCommand
+
+from flask_sqlalchemy import SQLAlchemy
+
+import psycopg2
+
+app = Flask(__name__)
+
+db_string = "postgres://xpcdulaqqancpa:0616d8130a659a869dacb11768aaf0d8f87f741b19fc8bf46d3af34c78f5ca85@ec2-34-200-72-77.compute-1.amazonaws.com:5432/ddjr78vh1cvcc4"
+app.config['SQLALCHEMY_DATABASE_URI'] = db_string
+app.config['SQLALCHEMY TRACK_MODIFICATIONS'] = False
+
+#db = create_engine(db_string)
+db = SQLAlchemy(app)
+base = declarative_base()
+
+migrate = Migrate(app, db)
+manager = Manager(app)
+
+manager.add_command('db', MigrateCommand)
+
+class Item(db.Model):
+  __tablename__ = 'Item'
+
+  __mapper_args__ = {
+    'confirm_deleted_rows': False
+  }
+
+  name = Column(String, primary_key=True)
+  price = Column(Float)
+  priceperunit = Column(Float)
+  store = Column(String)
+
+class store(db.Model):
+  __tablename__ = 'store'
```

Comment:
I'm used to having my SQL table names in `snake_case` and plural. We can use `PascalCase` and singular names, but let's make sure to stay consistent with whatever convention we choose. Right now, `Item` is PascalCase but `store` is `snake_case`